libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -112,12 +112,13 @@ const AP_Param::GroupInfo AP_BLHeli::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("POLES",  9, AP_BLHeli, motor_poles, 14),
 
-    // @Param: REMASK
-    // @DisplayName: BLHeli bitmask of reversible channels
-    // @Description: Mask of channels which are reversible. This is used for ESCs which have been configured in '3D' mode, allowing for the motor to spin in either direction
+    // @Param: D3MASK
+    // @DisplayName: BLHeli bitmask of 3D channels
+    // @Description: Mask of channels which are dynamically reversible. This is used to configure ESCs in '3D' mode, allowing for the motor to spin in either direction
     // @Bitmask: 0:Channel1,1:Channel2,2:Channel3,3:Channel4,4:Channel5,5:Channel6,6:Channel7,7:Channel8,8:Channel9,9:Channel10,10:Channel11,11:Channel12,12:Channel13,13:Channel14,14:Channel15,15:Channel16
     // @User: Advanced
-    AP_GROUPINFO("REMASK",  10, AP_BLHeli, channel_reversible_mask, 0),
```

Comment:
need a ETYPE param for ESC type, so we know if it is BLHeli32, Kiss, APD etc, default none

Reply:
Added SERVO_DSHOT_ESC